nautobot/core/runner/importer.py

```diff
@@ -67,7 +66,6 @@ def __repr__(self):
         return f"<{type(self)} for '{self.name}' ({self.config_path})>"
 
     def validate(self):
-        # TODO(dcramer): is there a better way to handle validation so it
```

Comment:
Feels like this is maybe still a TODO for us since we've vendored this code? If we agree that it isn't, then the following comment line should be removed as well?

Reply:
Yeah good call.